### ISSUE TITLE
v 3.0.1 Maven Central artifacts are corrupted -- zip END header not found

### ISSUE THREAD

**Issue by jgenoctr** [groovy/GMavenPlus#281]
Greeings. This issue is with v 3.0.1 using jdk-21.

Maven info:
Apache Maven 3.9.4 (dfbb324ad4a7c8fb0bf182e6d91b0ae20e3d2dd9)
Maven home: /Users/jgenoctr/opt/apache-maven-3.9.4
Java version: 21, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk-21.jdk/Contents/Home
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "13.4.1", arch: "x86_64", family: "mac"

```
Downloading from central: https://repo1.maven.org/maven2/org/codehaus/gmavenplus/gmavenplus-plugin/3.0.1/gmavenplus-plugin-3.0.1.pom
Downloaded from central: https://repo1.maven.org/maven2/org/codehaus/gmavenplus/gmavenplus-plugin/3.0.1/gmavenplus-plugin-3.0.1.pom (21 kB at 49 kB/s)
Downloading from central: https://repo1.maven.org/maven2/org/codehaus/gmavenplus/gmavenplus-plugin/3.0.1/gmavenplus-plugin-3.0.1.jar
Downloaded from central: https://repo1.maven.org/maven2/org/codehaus/gmavenplus/gmavenplus-plugin/3.0.1/gmavenplus-plugin-3.0.1.jar (21 kB at 496 kB/s
```

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.916 s
[INFO] Finished at: 2023-09-29T11:56:52-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to parse plugin descriptor for org.codehaus.gmavenplus:gmavenplus-plugin:3.0.1 (/Users/jgenoctr/.m2/repository/org/codehaus/gmavenplus/gmavenplus-plugin/3.0.1/gmavenplus-plugin-3.0.1.jar): zip END header not found -> [Help 1]
```
```
$:shasum -a 256 gmavenplus-plugin-3.0.1.jar
0233d15a64a86b5f695d8653aac8a4ce6becf701b101152e0451c6a2a96c09a8  gmavenplus-plugin-3.0.1.jar

$:shasum -a 256 gmavenplus-plugin-3.0.1.pom
0233d15a64a86b5f695d8653aac8a4ce6becf701b101152e0451c6a2a96c09a8  gmavenplus-plugin-3.0.1.pom
```

It appears that the Maven Repository itself is corrupt, and the POM and the jar are actually the same artifact.

**Comment by keeganwitt** [comment]
This has been fixed in 3.0.2.

**Comment by jgenoctr** [comment]
Confirmed: 
- issue fixed (both from Maven and Eclipse) in 3.0.2. 

Excellent work!
